### DISCUSSION TITLE
command remote_start_drive exception {"response":null,"error":"user not found","error_description":""}

### DISCUSSION THREAD

**Discussion by joinwangzi** [timdorr/tesla-api#556] [Q&A]
request: https://owner-api.teslamotors.com/api/1/vehicles/xxxxxxxx/command/remote_start_drive
response : {"response":null,"error":"user not found","error_description":""}

I am in China, It was normal for me to use the remote start command before March 26, 2022, and then I used the remote start command again on March 29, 2022 and found that it could not be used normally. It will return {"reason": "", "result": true}.

However, in recent days, it has been found that this API interface cannot be used normally. It returns {"response": null, "error": "user not found", "error_description": ""}.
I confirm that the ID and password are correct.
Other commands can be used normally, such as wake_up、honk_horn、flash_lights.

I tried to refresh the token and change the login password, but it still didn't work.
Is anyone in this situation? Please help me! Thanks

**Comment by joinwangzi** [comment]
Does anyone encounter this situation?

**Reply by dacastro4** [reply]
I got it multiple times. Did you make sure that the API token is valid and the vehicle ID is the correct one?
